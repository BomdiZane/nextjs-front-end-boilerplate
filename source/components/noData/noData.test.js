import NoData from './index';

describe('NoData component:', () => {
	const noData = shallow(<NoData  />);

	it('should match snapshot', () => expect(noData).toMatchSnapshot());
});
