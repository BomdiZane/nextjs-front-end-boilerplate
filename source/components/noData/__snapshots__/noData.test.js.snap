// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoData component: should match snapshot 1`] = `
<WithStyles(Typography)
  style={
    Object {
      "color": "crimson",
      "textAlign": "center",
      "width": "100%",
    }
  }
  variant="h5"
/>
`;
